### DISCUSSION TITLE
How to use CpModel.AddCumulative function with new 9.8.3296 OrTools C# version

### DISCUSSION THREAD

**Discussion by JharPaat** [google/or-tools#4104] [CP-SAT questions]
Hi my question same as previous issue. 
I would like to know how to use CpModel.AddCumulative function (with only 1 parameter) with new 9.8.3296 OrTools version.

https://github.com/google/or-tools/discussions/3753

Old question:

_Originally posted by @kevinl412 in https://github.com/google/or-tools/discussions/3753_

**Comment by lperron** [comment]
the 1 param version is not available in python.

**Reply by JharPaat** [reply]
Actually I am using C# Version. 
I have corrected the Title.
@Iperron Could you please provide some examples for C# version with 1 Parameter ?

public CumulativeConstraint AddCumulative<C>(C capacity)

**Comment by lperron** [comment]
I have no code.

Create the constraint.

https://or-tools.github.io/docs/dotnet/classGoogle_1_1OrTools_1_1Sat_1_1CpModel.html#ae0f6c541b665cf5d5458e3daed924d40


Add all demands 1 by 1.

https://or-tools.github.io/docs/dotnet/classGoogle_1_1OrTools_1_1Sat_1_1CumulativeConstraint.html

or all of them with the AddDemands
Laurent Perron | Operations Research | ***@***.*** | (33) 1 42 68 53
00
